py/embedded-server/deephaven_server/start_jvm.py

```diff
@@ -48,28 +49,35 @@ def _jars():
 
 # Provide a util func to start the JVM, will use its own defaults if none are offered
 def start_jvm(
-        jvm_args = None,
-        jvm_properties = None,
-        java_home = None,
-        extra_classpath = [],
-        propfile: str = None,
-        config = None):
+        jvm_args: Optional[List[str]] = None,
+        default_jvm_args: Optional[List[str]] = None,
+        jvm_properties: Optional[Dict[str, str]] = None,
+        java_home: Optional[str] = None,
+        extra_classpath: Optional[List[str]] = None,
+        prop_file: str = None,
+        config = None,
+) -> None:
     """ This function uses the default DH property file to embed the Deephaven server and starts a Deephaven Python
     Script session. """
-    jvm_args = jvm_args or DEFAULT_JVM_ARGS
+    default_jvm_args = default_jvm_args or DEFAULT_JVM_ARGS
+    jvm_args = default_jvm_args + jvm_args if jvm_args else default_jvm_args
```

Comment:
Are there any concerns about how this works?  It is possible to get multiple entries or possibly conflicting entries.  Practically, it may work in all reasonable cases, so it isn't worth wasting time on.

My thought is that it is good enough, but I'm raising the question.

Reply:
I'm at least not unhappy with how it aligns with our env var usage. There are some flags where you can't meaningfully repeat them (`-Xmx` etc), while others you can (`-classpath`), so I'm not sure how to draw that line without actually parsing them all.